lib/text/ui_text_displayer.js

```diff
@@ -983,10 +964,12 @@ shaka.text.UITextDisplayer = class {
    * @param {string} lengthValue
    * @param {!shaka.text.Cue} cue
    * @param {HTMLElement} videoContainer
+   * @param {number=} scaleFactor
    * @return {string}
    * @private
    */
-  static convertLengthValue_(lengthValue, cue, videoContainer) {
+  static convertLengthValue_(lengthValue, cue, videoContainer,
+      scaleFactor = 1) {
     const lengthValueInfo =
         shaka.text.UITextDisplayer.getLengthValueInfo_(lengthValue);
 
```

Comment:
If there is no size specified then the cues will have no scale factor applied to them and remain presumably small and potentially not readible.  Is this there place to default and hence then apply a scale factor?

Reply:
@matvp91 is going to work on it soon, he wants to be able to customize some things, so it will be coming soon!

Reply:
Great @avelad thanks.
@matvp91 please can you share your plans for customization?  I've got a vested interest in this area currently, maybe we can divide and conquer!